Scripts/Runtime/Entities/TaskSystem/AbstractTaskDriver.cs

```diff
@@ -110,5 +160,18 @@ internal List<CancelRequestsDataStream> GetSubTaskDriverCancelRequests()
 
 
         //TODO: #73 - Implement other job types
+        
+        //*************************************************************************************************************
+        // SAFETY
+        //*************************************************************************************************************
+        
+        [Conditional("ENABLE_UNITY_COLLECTIONS_CHECKS")]
```

Comment:
Maybe worth putting under `DEBUG` or `UNITY_ASSERTIONS` since it's not really a collections usage safety check is it?

Reply:
We had decided a while back to piggy back on `ENABLE_UNITY_COLLECTIONS_CHECKS` for all safety checks in the `anvil-unity-dots` library. 

The rationale being that we wanted our safety to be disabled and fast whenever we wanted Unity's safety code to be disabled and fast.

Reply:
OK cool. I don't remember that specifically for all assertions but I'm ok with that. Can we update the README at some point to mention that?

Reply:
Yes, added issue #90